README.md

```diff
@@ -555,6 +556,11 @@ A curated list of awesome mathematics resources.
 * [Mathematical Tools for Physics](http://www.physics.miami.edu/nearing/mathmethods/) - James Nearing
 * [Mechanism of the Heavens (1831)](http://www.malaspina.com/etext/heavens.htm) - Mary Somerville
 
+# Students Lecture Notes
+**(Aggregated List of Many Math Subjects)**
```

Comment:
Thanks for your contribution! I'd suggest to remove this line, but to add a few more details to the two lines below after the university: years (e.g. 2012-2018) and a couple of words on the topics covered (e.g. "elementary and advanced algebra", or "theoretical physics, statistics" etc)